src/backend/tools/index.ts

```diff
@@ -291,6 +291,44 @@ export const DXVK = {
           protonVerb: 'run'
         })
       })
+
+      logInfo('Removing DXVK DLLs', LogPrefix.DXVKInstaller)
+
+      // removing DXVK dlls
+      if (is64bitPrefix) {
+        dlls32.forEach((dll) => {
+          rm(`${winePrefix}/drive_c/windows/syswow64/${dll}`, (err) => {
+            if (err) {
+              logError([`Error removing ${dll}`, err], LogPrefix.DXVKInstaller)
+            }
+          })
+        })
+        dlls64.forEach((dll) => {
+          rm(`${winePrefix}/drive_c/windows/system32/${dll}`, (err) => {
+            if (err) {
+              logError([`Error removing ${dll}`, err], LogPrefix.DXVKInstaller)
+            }
+          })
+        })
+      } else {
+        dlls32.forEach((dll) => {
+          rm(`${winePrefix}/drive_c/windows/system32/${dll}`, (err) => {
+            if (err) {
+              logError([`Error removing ${dll}`, err], LogPrefix.DXVKInstaller)
+            }
+          })
+        })
+      }
```

Comment:
maybe this can be change to use the if/else stuff to set an array of paths to delete and then loop through the array to delete things?

something like:

```ts
const dllsPathsToDelete = dlls32.map((dll) => `${winePrefix}/drive_c/windows/system32/${dll}`)
if (is64bitPrefix) dllsPathsToDelete.concat(dlls64.map((dll) => `${winePrefix}/drive_c/windows/syswow64/${dll}`))

dllsPathsToDelete.forEach((dllPath) => {
    rm(dllPath, (err) => {
      if (err) {
        logError([`Error removing ${dllPath}`, err], LogPrefix.DXVKInstaller)
      }
    })
}
```

Reply:
I was just trying to change the original code as little as possible to reduce the chance of introducing a new bug but if you guys prefer to try to optimise/clean up a little bit in the same PR I can certainly look into changing this. Just let me know.

Reply:
my suggestion is to only refactor this new code that is added, not to change the rest of the code that was there